### DISCUSSION TITLE
How to download coordinates from cosmograph.app

### DISCUSSION THREAD

**Discussion by wakawaka111** [cosmograph-org/cosmos#116] [Q&A]
Is there any convenient way to download all the nodes' coordinates information from the website?

**Comment by rokotyan** [comment]
Check out our article on Nightingale, it has the instuctions https://nightingaledvs.com/how-to-visualize-a-graph-with-a-million-nodes/

> Another important thing that you might want to do is to export the layout, i.e. the coordinates of each node, which is a feature that is thankfully also available in Cosmograph. When you select a portion of or the whole graph with the Rectangular Selection tool, you’ll see the Save Current Layout switch under the Export Selected Data section. Turn it on, click on the Metadata button, and voila, the downloaded file will have additional x and y columns with the coordinates of each node.

**Comment by yizhe-ang** [comment]
Hello! May I ask how will this be possible under the current version of cosmograph?
